packages/manager-components/src/components/clipboard/clipboard.component.tsx

```diff
@@ -5,11 +5,13 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import './translations';
 
-export const Clipboard: React.FC<OdsClipboardAttribute> = (props) => {
+export const Clipboard: React.FC<OdsClipboardAttribute & {
```

Comment:
You should not modify a `manager-component` on a feature branch.
In your case, it may be easier to update your tests to rely only on the clipboard text value.